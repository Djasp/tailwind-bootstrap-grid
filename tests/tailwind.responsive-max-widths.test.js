const run = require('./run-test');

it('should handle responsive max widths', () => run(__filename));
