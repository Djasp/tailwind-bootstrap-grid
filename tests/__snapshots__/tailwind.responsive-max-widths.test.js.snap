// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should handle responsive max widths 1`] = `
".container {
  width: 100%;
  margin-right: auto;
  margin-left: auto;
  padding-right: 15px;
  padding-left: 15px;
}

@media (min-width: 60em) {
  .container {
    max-width: 20em;
  }
}

@media (min-width: 80em) {
  .container {
    max-width: 40em;
  }
}

@media (min-width: 120em) {
  .container {
    max-width: 60em;
  }
}

.container-fluid {
  width: 100%;
  padding-right: 15px;
  padding-left: 15px;
  margin-right: auto;
  margin-left: auto;
}

.row.no-gutters {
  margin-right: 0;
  margin-left: 0;
}

.row.no-gutters > .col,
.row.no-gutters > .mobile\\\\:col,
.row.no-gutters > .tablet\\\\:col,
.row.no-gutters > .desktop\\\\:col,
.row.no-gutters > [class*=\\"col-\\"],
.row.no-gutters > [class*=\\"mobile\\\\:col-\\"],
.row.no-gutters > [class*=\\"tablet\\\\:col-\\"],
.row.no-gutters > [class*=\\"desktop\\\\:col-\\"] {
  padding-right: 0;
  padding-left: 0;
}

.row {
  display: flex;
  flex-wrap: wrap;
  margin-left: -15px;
  margin-right: -15px;
}

.col,
.mobile\\\\:col,
.tablet\\\\:col,
.desktop\\\\:col,
.col-auto,
.mobile\\\\:col-auto,
.tablet\\\\:col-auto,
.desktop\\\\:col-auto,
.col-1,
.mobile\\\\:col-1,
.tablet\\\\:col-1,
.desktop\\\\:col-1,
.col-2,
.mobile\\\\:col-2,
.tablet\\\\:col-2,
.desktop\\\\:col-2,
.col-3,
.mobile\\\\:col-3,
.tablet\\\\:col-3,
.desktop\\\\:col-3 {
  position: relative;
  width: 100%;
  padding-right: 15px;
  padding-left: 15px;
}

.col {
  flex-basis: 0;
  flex-grow: 1;
  max-width: 100%;
}

.col-auto {
  flex: 0 0 auto;
  width: auto;
  max-width: 100%;
}

.col-1 {
  flex: 0 0 33.333333333333336%;
  max-width: 33.333333333333336%;
}

.col-2 {
  flex: 0 0 66.66666666666667%;
  max-width: 66.66666666666667%;
}

.col-3 {
  flex: 0 0 100%;
  max-width: 100%;
}

.order-first {
  order: -1;
}

.order-last {
  order: 4;
}

.order-0 {
  order: 0;
}

.order-1 {
  order: 1;
}

.order-2 {
  order: 2;
}

.order-3 {
  order: 3;
}

.offset-0 {
  margin-left: 0%;
}

.offset-1 {
  margin-left: 33.333333333333336%;
}

.offset-2 {
  margin-left: 66.66666666666667%;
}

@media (min-width: 60em) {
  .mobile\\\\:col {
    flex-basis: 0;
    flex-grow: 1;
    max-width: 100%;
  }

  .mobile\\\\:col-auto {
    flex: 0 0 auto;
    width: auto;
    max-width: 100%;
  }

  .mobile\\\\:col-1 {
    flex: 0 0 33.333333333333336%;
    max-width: 33.333333333333336%;
  }

  .mobile\\\\:col-2 {
    flex: 0 0 66.66666666666667%;
    max-width: 66.66666666666667%;
  }

  .mobile\\\\:col-3 {
    flex: 0 0 100%;
    max-width: 100%;
  }

  .mobile\\\\:order-first {
    order: -1;
  }

  .mobile\\\\:order-last {
    order: 4;
  }

  .mobile\\\\:order-0 {
    order: 0;
  }

  .mobile\\\\:order-1 {
    order: 1;
  }

  .mobile\\\\:order-2 {
    order: 2;
  }

  .mobile\\\\:order-3 {
    order: 3;
  }

  .mobile\\\\:offset-0 {
    margin-left: 0%;
  }

  .mobile\\\\:offset-1 {
    margin-left: 33.333333333333336%;
  }

  .mobile\\\\:offset-2 {
    margin-left: 66.66666666666667%;
  }
}

@media (min-width: 80em) {
  .tablet\\\\:col {
    flex-basis: 0;
    flex-grow: 1;
    max-width: 100%;
  }

  .tablet\\\\:col-auto {
    flex: 0 0 auto;
    width: auto;
    max-width: 100%;
  }

  .tablet\\\\:col-1 {
    flex: 0 0 33.333333333333336%;
    max-width: 33.333333333333336%;
  }

  .tablet\\\\:col-2 {
    flex: 0 0 66.66666666666667%;
    max-width: 66.66666666666667%;
  }

  .tablet\\\\:col-3 {
    flex: 0 0 100%;
    max-width: 100%;
  }

  .tablet\\\\:order-first {
    order: -1;
  }

  .tablet\\\\:order-last {
    order: 4;
  }

  .tablet\\\\:order-0 {
    order: 0;
  }

  .tablet\\\\:order-1 {
    order: 1;
  }

  .tablet\\\\:order-2 {
    order: 2;
  }

  .tablet\\\\:order-3 {
    order: 3;
  }

  .tablet\\\\:offset-0 {
    margin-left: 0%;
  }

  .tablet\\\\:offset-1 {
    margin-left: 33.333333333333336%;
  }

  .tablet\\\\:offset-2 {
    margin-left: 66.66666666666667%;
  }
}

@media (min-width: 120em) {
  .desktop\\\\:col {
    flex-basis: 0;
    flex-grow: 1;
    max-width: 100%;
  }

  .desktop\\\\:col-auto {
    flex: 0 0 auto;
    width: auto;
    max-width: 100%;
  }

  .desktop\\\\:col-1 {
    flex: 0 0 33.333333333333336%;
    max-width: 33.333333333333336%;
  }

  .desktop\\\\:col-2 {
    flex: 0 0 66.66666666666667%;
    max-width: 66.66666666666667%;
  }

  .desktop\\\\:col-3 {
    flex: 0 0 100%;
    max-width: 100%;
  }

  .desktop\\\\:order-first {
    order: -1;
  }

  .desktop\\\\:order-last {
    order: 4;
  }

  .desktop\\\\:order-0 {
    order: 0;
  }

  .desktop\\\\:order-1 {
    order: 1;
  }

  .desktop\\\\:order-2 {
    order: 2;
  }

  .desktop\\\\:order-3 {
    order: 3;
  }

  .desktop\\\\:offset-0 {
    margin-left: 0%;
  }

  .desktop\\\\:offset-1 {
    margin-left: 33.333333333333336%;
  }

  .desktop\\\\:offset-2 {
    margin-left: 66.66666666666667%;
  }
}
"
`;
